src/js/views/createCertificate/translations/pt_br.createCertificate.i18n.json

```diff
@@ -0,0 +1,36 @@
+{
+  "headerTitle": "Novo Certificado",
+  "finishButton": "Finalizar",
+  "cancelButton": "Cancelar",
+  "createCertificateOneClick": {
+    "title": "Criar certificado com um clique",
+    "subTitle": "Mais prático: A plataforma gera o certificado e a chave privada para o dispositivo"
+  },
+  "createCertificateCSR": {
+    "title": "Submeter uma Solicitação de Assinatura de Certificado (CSR)",
+    "subTitle": "Mais seguro: A plataforma gera um certificado, mas não tem conhecimento da chave privada.",
```

Comment:
mas sem ter conhecimento da sua chave privada* Maybe sounds more informative (the same in English)